easybuild/scripts/bootstrap_eb.py

```diff
@@ -68,7 +76,9 @@
 
 VSC_BASE = 'vsc-base'
 VSC_INSTALL = 'vsc-install'
-EASYBUILD_PACKAGES = [VSC_INSTALL, VSC_BASE, 'easybuild-framework', 'easybuild-easyblocks', 'easybuild-easyconfigs']
+# Python 3 is not supported by the vsc-* packages
+EASYBUILD_PACKAGES = (([] if IS_PY3 else [VSC_INSTALL, VSC_BASE]) +
```

Comment:
@boegel I had the same frustrating to-and-fro with the hound last week, can we disable this warning?

Reply:
The warning is actually caused by the PR adding the "black would make changes" ignore rule which overwrites standard ignore which has those 2 warnings included. This would also have been detected if Flake8 checks were run on CI over all source files, see https://github.com/easybuilders/easybuild-easyblocks/issues/1932

Reply:
fixed in #3193